Department-for-Transport-Disruptions/bods-integrated-data
# BODS-3516: Mock data producer

To test locally:
- Run `make setup` (think this should install tflocal - if not install tflocal on your machine)
- Run `make create-avl-mock-data-producer`
- In Localstack desktop, check the following:
1. Go to Dynamo, check in subscriptions table that the mock data producer subscription has been created (note the PK which is the subscription ID)
2. Go to S3, to `integrated-data-siri-vm-local` bucket. There should be a folder with the name of your subscription ID, check if some avl data is stored in there. (Send data lambda runs once a minute so may take a minute to come through - you can check logs of `avl-mock-data-producer-send-data-local` lambda)
